chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -256,6 +249,7 @@ public class Scene implements JsonSerializable, Refreshable {
   /** Controls how much the fog color is blended over the sky/skymap. */
   protected double skyFogDensity = 1;
 
+  protected boolean using3dBiomes = false;
```

Comment:
```suggestion
  protected boolean use3dBiomes = false;
```

Also, remember to save and load that value from the scene description :bulb: